pom.xml

```diff
@@ -146,16 +146,17 @@
         <commons-lang.version>2.6</commons-lang.version>
         <commons-lang3.version>3.4</commons-lang3.version>
         <commons.version>2.5</commons.version>
-        <curator.version>2.11.0</curator.version>
-        <dropwizard.version>1.0.5</dropwizard.version>
+        <curator.version>4.0.1</curator.version>
+        <curator.test.version>2.12.0</curator.test.version>
```

Comment:
is it ok to have different version just for test ?

Reply:
I picked it up from Registry project, but it doesn't look like needed for Streamline. Will remove.